GerPS-onto/.htaccess

```diff
@@ -0,0 +1,4 @@
+# Turn off MultiViews
+Options -MultiViews
+
+Redirect 301 /GerPS-onto /gerps/ontology/process
```

Comment:
This would go to the same id twice: `w3id.org/GerPS-onto//GerPS-onto`. Please review

Reply:
I think this is ok since it's `Redirect`.